smash/tests/core/simulation/test_estimate.py

```diff
@@ -19,50 +19,55 @@ def generic_multiset_estimate(model: smash.Model, **kwargs) -> dict:
         "bounds": [(20, 1000), (-20, 5), (0.2, 0.8)],
     }
 
-    sample = smash.factory.generate_samples(problem, n=30, random_state=11)
+    sample = smash.factory.generate_samples(problem, n=12, random_state=1)
 
-    multisets = {}
+    multiset_1 = smash.multiple_forward_run(model, sample, common_options={"ncpu": ncpu, "verbose": False})
 
-    multisets["mfwr"] = smash.multiple_forward_run(
-        model, sample, common_options={"ncpu": ncpu, "verbose": False}
-    )
+    spl_sample = {k: [] for k in problem["names"]}
+    for i in range(sample.n_sample):
+        instance = model.copy()
 
-    multisets["mopt_unf"] = smash.multiple_optimize(
-        model,
-        sample,
-        mapping="uniform",
-        optimize_options={"termination_crit": {"maxiter": 1}},
-        common_options={"ncpu": ncpu, "verbose": False},
-    )
+        for k in problem["names"]:
+            try:
+                instance.set_rr_parameters(k, getattr(sample, k)[i])
+            except ValueError:
+                instance.set_rr_initial_states(k, getattr(sample, k)[i])
+
+        instance.optimize(
+            mapping="uniform",
+            optimize_options={"parameters": problem["names"], "termination_crit": {"maxiter": 1}},
+            common_options={"ncpu": ncpu, "verbose": False},
+        )
 
-    multisets["mopt_ml"] = smash.multiple_optimize(
-        model,
-        sample,
-        mapping="multi-linear",
-        optimize_options={"termination_crit": {"maxiter": 1}},
-        common_options={"ncpu": ncpu, "verbose": False},
+        for k in problem["names"]:
+            try:
+                value = instance.get_rr_parameters(k).copy()
+            except ValueError:
+                value = instance.get_rr_initial_states(k).copy()
```

Comment:
pourquoi try / except ? Je prefere un ``if k in instance.rr_parameters.keys`` truc comme ca